charts/quantum-serverless/charts/gateway/values.yaml

```diff
@@ -126,7 +126,6 @@ ingress:
   enabled: false
   className: ""
```

Comment:
@psschwei do we want to leave this value empty here to be dynamically populated, or set `nginx` as default?

Reply:
Let's default to `nginx`, as that's what we used on the old annotation